### PR TITLE
backend/jupiterbrain: add context to all errors

### PR DESCRIPTION
**Pull request by henrikhodne**
This uses github.com/pkg/errors a lot to provide some context for all errors, which makes it easier for us to figure out exactly where an error comes from when we see an error in the logs, instead of just "file not found" or "EOF".

### CODE REVIEW

**Comment by meatballhat** [comment]
Is there an update for `vendor/manifest` or were we already using this package?

Otherwise :sheep: :it:

**Comment by henrikhodne** [comment]
> Is there an update for vendor/manifest or were we already using this package?

We were already using it, but I just pushed an upgrade to update to 0.7.1 (and thereby also locking it to a released version instead of some in-between SHA).

File: vendor/manifest [hunk 1]
```diff
@@ -295,8 +295,8 @@
 			"importpath": "github.com/pkg/errors",
 			"repository": "https://github.com/pkg/errors",
 			"vcs": "git",
-			"revision": "a2d6902c6d2a2f194eb3fb474981ab7867c81505",
-			"branch": "master",
+			"revision": "17b591df37844cde689f4d5813e5cea0927d8dd2",
+			"branch": "HEAD",
```
Comment (meatballhat) [inline vendor/manifest]: the most exciting branch of all

**Comment by meatballhat** [comment]
:shipit:

**Pull request merged**